native/exe/MiniClientLauncher/MiniClientLauncher.cpp

```diff
@@ -149,8 +149,8 @@ int APIENTRY WinMain(HINSTANCE hInstance,
 	HMODULE exeMod = GetModuleHandle(NULL);
 	LPTSTR appPath = new TCHAR[512];
 	GetModuleFileName(exeMod, appPath, 512);
-	int appLen = strlen(appPath);
-	for (int i = appLen - 1; i > 0; i--)
+	size_t appLen = strlen(appPath);
+	for (size_t i = appLen - 1; i > 0; i--)

```

Comment:
This is dangerous if appLen is zero...so handle that case too.

Reply:
Think I have this fixed appropriately